### PR TITLE
Update smashbox documentation with quickstart, description of important tests and monitoring integration

### PR DESCRIPTION
**Pull request by mrow4a**
As in the title

selection_126

@patrickjahns @DeepDiver1975 @PVince81

### CODE REVIEW

**Comment by mrow4a** [comment]
Updated requirements for this PR to be merged

**Comment by tboerger** [comment]
Conflicts...

**Comment by mrow4a** [comment]
Yep, and will be, I need to update "TODOs" and merge other related PR to merge the documentation here

**Comment by PVince81** [comment]
finish this ?

**Comment by mrow4a** [comment]
Will need to discuss with @tboerger perf dashboard.

**Comment by mrow4a** [comment]
Please review the new docs and merge

**Comment by patrickjahns** [comment]
@mrow4a 
stray commit? https://github.com/owncloud/smashbox/pull/154/files#diff-6b98094ef0e436e233b8e98a607f0b6a

File: README.md [hunk 1]
```diff
@@ -21,7 +21,80 @@ The goal of this is to:
...
+- Check sync client directories of workers
+```
+$ ls ~/smashdir/test_nplusone/worker1/
+```
+
+Important integration tests
+===========================
+
+ * [Basic Sync and Conflicts ](lib/test_basicSync.py)
+    - basicSync_filesizeKB from 1kB to 50MB (normal and chunked files sync)
```
Comment (patrickjahns) [inline README.md:59]: is this configurable?

**Comment by mrow4a** [comment]
@patrickjahns the change in tests was on purpose

File: integration_test_cmds.sh [hunk 2]
```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
```
Comment (mrow4a) [inline integration_test_cmds.sh]: ok, sorry, this actually sneaked in - we dont need it

File: lib/test_nplusone.py [hunk 3]
```diff
@@ -21,32 +21,22 @@
...
-        },
-
-        { 'nplusone_filesize': OWNCLOUD_CHUNK_SIZE(0.3), 
-          'nplusone_nfiles':10
-        },
-
-        { 'nplusone_filesize': OWNCLOUD_CHUNK_SIZE(1.3), 
-          'nplusone_nfiles':2
+        { 'nplusone_filesize': OWNCLOUD_CHUNK_SIZE(0.1),
+          'nplusone_nfiles':100,
```
Comment (mrow4a) [inline lib/test_nplusone.py:25]: removed some tests as they were redundant (to other tests)

File: README.md (hunk 1 above)
Comment (mrow4a) [reply README.md:59]: yes, basic sync can be fonfigurable by flags

**Pull request merged**